releasenotes/notes/preprocessing-full-dependency-2a62f62e36d5ace6.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Think there also needs to be an `upgrade` section that mentions that we removed the `preprocessing` extra install arg

Reply:
Added! But on second thought -> is this an upgrade or a deprecation?

Reply:
Deprecations are for things that will continue to work for a while. In this case because we're just removing it, it's an upgrade (IMO).